Makefile

```diff
@@ -13,7 +13,11 @@ TARGETS := $(shell ls scripts)
 	fi
 
 $(TARGETS): .dapper
-	./.dapper $@
+	@if [[ "$@" = "post-release-checks" ]] || [[ "$@" = "list-gomod-updates" ]] || [[ "$@" = "check-chart-kdm-source-values" ]]; then\
```

Comment:
Does the `check-chart-kdm-source-values` script need to be added to this PR or is it in another one?

Reply:
Nevermind it's here https://github.com/rancher/rancher/pull/37795/files